pbxproj/pbxextensions/ProjectFiles.py

```diff
@@ -337,7 +337,7 @@ def remove_files_by_path(self, path, tree=TreeType.SOURCE_ROOT, target_name=None
 
         return result != 0 and result == total
 
-    def add_folder(self, path, parent=None, excludes=None, recursive=True, create_groups=True, target_name=None,
+    def add_folder(self, path, parent=None, excludes=None, is_group_abspath=True, recursive=True, create_groups=True, target_name=None,
```

Comment:
rather than add a new parameter, it would be better to add a new FileOptions, for example, `self.add_groups_relative=False`, and use that option for the checks in line 372